src/core/Akka.Remote.Tests/RemotingTerminatorSpecs.cs

```diff
@@ -68,5 +70,37 @@ public void RemotingTerminator_should_shutdown_promptly_with_some_associations()
                 Assert.True(terminationTask.Wait(RemainingOrDefault), "Expected to terminate within 10 seconds, but didn't.");
             });
         }
+
+        [Fact]
+        public void RemotingTerminator_should_shutdown_properly_with_remotely_deployed_actor()
```

Comment:
This spec helped me reproduce a bug with the `RemoteDeploymentWatcher` where we weren't properly sending clearing out dead remotely deployed actors. Passes now.